SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005833	Wild Shot
+SKILL_20151224_005834	{#DD5500}{ol}[Missile]{/}{/}{nl}Attack simultaneously with crossbow and pistol held in both hands. 
+SKILL_20151224_005835	Sonic Strike
+SKILL_20151224_005836	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Quickly descend the hawk to attack the enemies in the designated area.
+SKILL_20151224_005837	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire cannonballs to target the enemies on ground. The damage is halved against airborne enemies.
+SKILL_20151224_005838	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire cannonballs to target airborne enemies. The damage is halved against enemies on ground.
+SKILL_20151224_005839	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire a howitzer to the ground. 5x damage is dealt to enemy's structure.
+SKILL_20151224_005840	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire a cannonball that spread to target all enemies in front of you. The user can be pushed back from the recoil of the shot.
+SKILL_20151224_005841	Attack: 150% + #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%
+SKILL_20151224_005842	{#DD5500}{ol}[Missile]{/}{/}{nl}Attack multiple enemies by firing at a designated area.
+SKILL_20151224_005843	{#DD5500}{ol}[Missile]{/}{/}{nl}Accurately aim at the head of the enemy to attack. The attacked enemy will have their intelligence and spirit temporality reduced.
+SKILL_20151224_005844	Attack: 130% + #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}Intelligence and Spirit reduction: #{CaptionRatio2}#
+SKILL_20151224_005845	{#DD5500}{ol}[Missile]{/}{/}{nl}Aim and fire at a far away enemy. The usual range of fire is increased and heavily damage a single enemy.
+SKILL_20151224_005846	Attack: 150% + #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionRatio}#%{nl}
+SKILL_20151224_005847	{#DD5500}{ol}[Missile]{/}{/}{nl}Fire continuously in a straight line in front of you until the maximum range of fire. The attacked enemies might be pushed back.
+SKILL_20151224_005848	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Use the butt of the gun to attack an enemy in close distance.
+SKILL_20151224_005849	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Stab an enemy by using the bayonet equipped to the rifle. The stabbed enemy might be pushed back.
+SKILL_20151224_005850	Attack: #{SkillAtkAdd}#{nl}Attribute Attack: #{CaptionTime}#%{nl}Recover 5% of Maximum HP{nl}Additional HP recovery: #{CaptionRatio}# ~ #{CaptionRatio2}#{nl}Magic Circle duration: 40 second
+SKILL_20151224_005851	Application count: #{CaptionRatio}#{nl}Duration: 60 second
+SKILL_20151224_005852	HP Recovery increase: #{CaptionRatio}#{nl}HP Recovery time reduction: #{CaptionRatio2}#second{nl}Torch duration: #{CaptionTime}#second
+SKILL_20151224_005853	Increase the count of effect for you and your party members.
+SKILL_20151224_005854	Maximum count increase: #{CaptionRatio}#{nl}Duration: #{CaptionTime}#second
+SKILL_20151224_005855	Lure the zombies by using the skull of a monkey and attack enemies for a period of time.
```

Comment:
Lure the zombies by using the skull of a monkey to attack enemies for a period of time.